### PR TITLE
IRS demo fixes

### PR DESCRIPTION
**Pull request by andr3ej**
* Increase max network map request size so the notary can register
* Suppress oracle service installation errors in non-oracle nodes
* Make demos automatically build capsule jars

### CODE REVIEW

File: node/src/main/kotlin/net/corda/node/internal/AbstractNode.kt [hunk 1]
```diff
@@ -268,16 +268,30 @@ abstract class AbstractNode(open val configuration: NodeConfiguration,
...
-            } catch (e: Exception) {
-                log.error("Unable to install Corda service ${it.name}", e)
-                null
-            }
+            tryInstallCordaService(it)
+        }
+    }
+
+    private fun <T : SerializeAsToken> tryInstallCordaService(serviceClass: Class<T>): T? {
+        val typeField = try {
```
Comment (Clintonio) [inline node/src/main/kotlin/net/corda/node/internal/AbstractNode.kt]: Can you add a TODO here related to the work Shams may be doing? (as discussed in person)

File: samples/irs-demo/src/main/kotlin/net/corda/irs/api/NodeInterestRates.kt [hunk 2]
```diff
@@ -95,6 +93,11 @@ object NodeInterestRates {
         )
         // DOCEND 3
 
+        companion object {
```
Comment (Clintonio) [inline samples/irs-demo/src/main/kotlin/net/corda/irs/api/NodeInterestRates.kt]: Have you checked the docs for any references to this?

File: node/src/main/kotlin/net/corda/node/internal/AbstractNode.kt [hunk 3]
```diff
@@ -268,16 +268,30 @@ abstract class AbstractNode(open val configuration: NodeConfiguration,
...
+            serviceClass.getField("type")
```
Comment (andr3ej) [inline node/src/main/kotlin/net/corda/node/internal/AbstractNode.kt]: Yes in the CordaService kdoc.

**Comment by mnesbit** [comment]
The fix for the network map message size is absolutely critical. I suggest we get this in ASAP. Not too keen on the gradle change, but I admit there probably isn't a simpler fix right now.

File: samples/irs-demo/src/main/kotlin/net/corda/irs/api/NodeInterestRates.kt (hunk 2 above)
Comment (andr3ej) [inline samples/irs-demo/src/main/kotlin/net/corda/irs/api/NodeInterestRates.kt]: Can't find any.

File: node/src/main/kotlin/net/corda/node/internal/AbstractNode.kt (hunk 1 above)
Comment (andr3ej) [inline node/src/main/kotlin/net/corda/node/internal/AbstractNode.kt]: Done

**Pull request merged**